internals/secrethub/rm.go

```diff
@@ -87,8 +87,8 @@ func (cmd *RmCommand) Run() error {
 
 	// Check if the secret exists first so we can return a generic error here instead of ErrSecretNotFound.
 	_, err = client.Secrets().Get(secretPath.Value())
-	if api.IsErrNotFound(err) {
-		return ErrResourceNotFound(cmd.path)
+	if err != nil {
+		return err
```

Comment:
Only not found errors used to be checked and returned here. Was this intentional and can it be changed?

Reply:
This was intentional and is explained in the comment above:

>  	// Check if the secret exists first so we can return a generic error here instead of ErrSecretNotFound. 

The remove command doesn't know whether the provided path points to a secret. Therefore, if no secret can be found on the given path a more generic error should be returned.

Reply:
Oh, thanks I didn't understand that comment at first. In this case we should still return `ErrResourceNotFound`, right?

Reply:
Yes exactly :+1: